### PR TITLE
Use google.golang.org/api/pubsub/v1 instead of cloud.google.com/go/pubsub

### PR DESCRIPTION
**Pull request by akm**
- Change an interface for Process
    - Use `pubsub.ReceivedMessage`
- Change lots of implementations 
- Update glide.yaml
- `go fmt`
- Add `.travis.yml`

### CODE REVIEW

File: process.go [hunk 1]
```diff
@@ -50,16 +49,20 @@ func (p *Process) execute(ctx context.Context) error {
...
 		msg_id := m.Attributes["job_message_id"]
 		progress, err := strconv.Atoi(m.Attributes["progress"])
 		if err != nil {
-			fmt.Println("Failed to convert %v to int message_id: %v cause of %v", m.Attributes["progress"], msg_id, err)
+			fmt.Printf("Failed to convert %v to int message_id: %v cause of %v", m.Attributes["progress"], msg_id, err)
 			return err
 		}
+		// https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage
+		// A timestamp in RFC3339 UTC "Zulu" format, accurate to nanoseconds. Example: "2014-10-02T15:01:23.045123456Z".
+		time, err := time.Parse(time.RFC3339, m.PublishTime)
```
Comment (minimum2scp) [inline process.go]: Variable name `time` conflicts with package name `time`, so I think it is better to change variable name (e.g.  `publishTime`)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+go:
+  - 1.6.3
+addons:
+  apt:
+    sources:
+      - sourceline: 'ppa:masterminds/glide'
+    packages:
+      - glide
+before_install:
+  ## Install glide
```
Comment (minimum2scp) [inline .travis.yml]: Unnecessary comment (`glide` is installed by addons > apt section)

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: go
+go:
+  - 1.6.3
```
Comment (minimum2scp) [inline .travis.yml]: Go 1.6.3 is a bit old.
I think 1.6.4 or 1.7.5, 1.8 is better.
Please refer https://golang.org/doc/devel/release.html

File: process.go (hunk 1 above)
Comment (akm) [inline process.go]: @minimum2scp Great suggestion! I fixed it.

File: .travis.yml (hunk 3 above)
Comment (akm) [inline .travis.yml]: @minimum2scp  Thanks. I chose 1.6.4 to avoid confusion with other GAE app.

File: .travis.yml (hunk 2 above)
Comment (akm) [inline .travis.yml]: @minimum2scp That's right. I removed it.

**Pull request merged**